app/src/main/java/com/willowtree/vocable/settings/EditPhrasesFragment.kt

```diff
@@ -71,7 +73,7 @@ class EditPhrasesFragment: BaseFragment() {
 
             phraseView.actionButtonContainer.deleteSayingsButton.action = {
                 if (phraseView.phraseEditText.tag is Phrase) {
-                    editPhrasesViewModel.deletePhrase(phraseView.phraseEditText.tag as Phrase)
+                    showDeletePhraseDialog(phraseView)
```

Comment:
You can actually just pass the `Phrase` object directly here instead of the view. You still have access to the `phrase` from line 56. I think this means we don't actually have to set it as a tag on the view either.